man/systemd.service.xml

```diff
@@ -268,15 +268,15 @@
             <varname>Type=</varname>, exits. Consequently, it cannot be used with
             <varname>Type=</varname><option>oneshot</option>.</para></listitem>
 
-            <listitem><para>The <option>cgroup</option> exit type is meant for applications whose forking model is not
-            known ahead of time and which might not have a specific main process. The service will stay running as long
-            as at least one process in the cgroup is running.</para></listitem>
+            <listitem><para>If set to <option>cgroup</option>, the service will be considered running as long as at
+            least one process in the cgroup has not exited.</para></listitem>
           </itemizedlist>
 
           <para>It is generally recommended to use <varname>ExitType=</varname><option>main</option> when a service has
           a known forking model and a main process can reliably be determined. <varname>ExitType=</varname>
-          <option>cgroup</option> is well suited for transient or automatically generated services, such as graphical
-          applications inside of a desktop environment.</para>
+          <option>cgroup</option> is meant for applications whose forking model is not known ahead of time and which
+          might not have a specific main process. It is well suited for transient or automatically generated services,
+          such as graphical applications inside of a desktop environment.</para>
```

Comment:
This whole chunk belongs in the previous commit. I'll force push to move it.

Reply:
Hmm, I can't push to your branch. Not sure why. Please move this hunk to the previous patch.

Reply:
Oops, should be fixed now